api/server/router/volume/volume.go

```diff
@@ -125,6 +133,7 @@ func (r *router) initRoutes() {
 				schema.VolumeAttachRequestSchema,
 				schema.VolumeSchema,
 				func() interface{} { return &apihttp.VolumeAttachRequest{} }),
+			handlers.NewPostArgsHandler(),
```

Comment:
A get route won't have post args, so why add it? If this is supposed to be an attach request, that already exists in this file.

```
		// attach an existing volume
		httputils.NewPostRoute(
			"volumeAttach",
			"/volumes/{service}/{volumeID}",
			r.volumeAttach,
			handlers.NewServiceValidator(),
			handlers.NewSchemaValidator(
				schema.VolumeAttachRequestSchema,
				schema.VolumeSchema,
				func() interface{} { return &apihttp.VolumeAttachRequest{} }),
		).Queries("attach"),
```